src/components/player-info.js

```diff
@@ -66,19 +84,133 @@ AFRAME.registerComponent("player-info", {
     deregisterComponentInstance(this, "player-info");
   },
   play() {
+    ["happy", "sad", "eww", "disgust", "angry", "smile", "hearts", "surprise", "clean"].map(x =>
+      this.el.sceneEl.addEventListener(x, this.emojiAction(x))
+    );
```

Comment:
Instead of emitting and listening to a different event name for each emoji, I would suggest emitting a single `change_emoji` with event details and listening to it with a single handler:
```javascript
this.el.sceneEl.addEventListener("change_emoji", this.changeEmoji)
...
changeEmoji(event) {
  const emojiType = event.detail.emojiType;
  const emojiImage = emojiTypeToImage[emojiType];
  ...
}
```